Utilities/File.cpp

```diff
@@ -229,46 +232,20 @@ bool fs::create_dir(const std::string& dir)
 
 bool fs::create_path(const std::string& path)
 {
-	g_tls_error = fse::ok;
```

Comment:
What's the point of that anyway? I only see it set to fse::ok, but never used anywhere.